clients/src/main/java/org/apache/kafka/common/errors/OffsetOutOfRangeException.java

```diff
@@ -35,4 +43,8 @@ public OffsetOutOfRangeException(String message, Throwable cause) {
         super(message, cause);
     }
 
+    public Map<TopicPartition, Long> getOutOfRangePartitions() {
```

Comment:
In Kafka, we tend to write the method name without the `get`, ie `outOfRangePartitions()`. The field name should have the same name though.